### DISCUSSION TITLE
Add pull requests to the result to the API

### DISCUSSION THREAD

**Discussion by pangolp** [community/community#112571] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

I am currently using the API, to list all the repositories of an organization, and I wanted to be able to filter/sort by pull request quantity. In such a way, we can try to close them as soon as possible, or at least try to make the number as close to zero as possible. So it would really be very useful to be able to filter by the number of open pull requests.

**Comment by nadila2k** [comment]
response #20583

**Comment by pangolp** [comment]
> response #20583

Thanks, I'll check it out. Sorry, I'm bad at searching sometimes.
